app/models/vm_or_template.rb

```diff
@@ -30,6 +30,9 @@ class VmOrTemplate < ApplicationRecord
   include TenancyMixin
 
   include AvailabilityMixin
+  include SupportsFeatureMixin
```

Comment:
@jameswnl I also just saw, that this is already included on line 8 :/
So you can remove that here